frontend/pages/hosts/HostDetailsPage/HostDetailsPage.jsx

```diff
@@ -665,6 +667,33 @@ export class HostDetailsPage extends Component {
         </div>
       );
     };
+
+    const renderDiskSpace = () => {
+      if (
+        host.gigs_disk_space_available !== 0 ||
```

Comment:
If the type is a number, let's always use `>` and `<` rather than `!`.

Reply:
Thank you!